plugin/service/storage/const.go

```diff
@@ -7,6 +7,9 @@ const (
 	// ConstAwsEndpointUrl is the key for the endpoint url in the aws s3 client.
 	ConstAwsEndpointUrl = "endpoint_url"
 
+	// ConstAwsDualStack is the key for the dualstack flag in the aws s3 client.
+	ConstAwsDualStack = "dual_stack"
```

Comment:
needs confirmation from @louisruch

Reply:
Not sure what I need to confirm, but one thing I was wondering here is if we should have an `internal` to other top level `const`/`globals` instead of defining the same const in two places.